zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/validator/AuthorizationRequestValidator.java

```diff
@@ -11,27 +11,30 @@
 import static io.camunda.zeebe.gateway.rest.validator.RequestValidator.validate;
 
 import io.camunda.zeebe.gateway.protocol.rest.AuthorizationRequest;
+import io.camunda.zeebe.protocol.record.value.AuthorizationScope;
 import java.util.Optional;
+import java.util.regex.Pattern;
 import org.springframework.http.ProblemDetail;
 
 public final class AuthorizationRequestValidator {
 
   public static Optional<ProblemDetail> validateAuthorizationRequest(
-      final AuthorizationRequest request) {
+      final AuthorizationRequest request, final Pattern idPattern) {
```

Comment:
❓ is this ever anything different to the configured pattern? I can't immediately see if it would have a use case of being different. If its never different we could just use the configured value in code for example

```
SecurityConfiguration.getIdValidationPattern()
```